CHANGELOG.md

```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.6.0] - 2021-07-08
+### Added
+* 10 new transforms, a masked loss wrapper, and a network adaptor for transfer learning
```

Comment:
Could you please change to `adapter` as the class name is `NetAdapter`? https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/netadapter.py